variables.tf

```diff
@@ -206,6 +206,12 @@ variable "http_proxy_url" {
   default     = ""
 }
 
+variable "additional_no_proxy" {
+  type        = string
```

Comment:
would it make more sense to do this as a list type and use [a join function](https://www.terraform.io/docs/configuration/functions/join.html) in the template file usage?

* [L35](https://github.com/hashicorp/terraform-aws-terraform-enterprise/pull/56/files#diff-264131a8410d0f8f58af1c713d120c53R35)
* [L65](https://github.com/hashicorp/terraform-aws-terraform-enterprise/pull/56/files#diff-264131a8410d0f8f58af1c713d120c53R65)
* [L101](https://github.com/hashicorp/terraform-aws-terraform-enterprise/pull/56/files#diff-264131a8410d0f8f58af1c713d120c53R101)

Reply:
Hmmm, yeah I think so! Especially since wrong formatting of the list can actually mess up some implementations of `no-proxy`.

Reply:
That is to say: good looking out! Thank you!